spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsProperties.java

```diff
@@ -133,6 +135,20 @@ public void setMaxUriTags(int maxUriTags) {
 			 */
 			private boolean autoTimeRequests = true;
 
+			/**
+			 * Default percentiles when {@link #autoTimeRequests} is {@code true} and
+			 * {@link Timed} is not presented on the corresponding request handler. Any
+			 * {@link Timed} annotation presented will have precedence.
+			 */
+			private double[] autoTimeRequestsDefaultPercentiles;
+
+			/**
+			 * Default histogram when {@link #autoTimeRequests} is {@code true} and
+			 * {@link Timed} is not presented on the corresponding request handler. Any
+			 * {@link Timed} annotation presented will have precedence.
+			 */
+			private boolean autoTimeRequestsDefaultHistogram;
```

Comment:
The field javadoc is used to document the configuration property in the reference documentation appendix and in the metadata that's used by IDEs. As such, it can't use javadoc tags and must be written in plain text. Can you please rework the description to match the style of the other fields in the class?